CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ if(FIND_PM_DEPS)
   find_package(ArgumentParser CONFIG REQUIRED)
   find_package(SwiftDriver CONFIG REQUIRED)
   find_package(SwiftCollections CONFIG REQUIRED)
+  find_package(SwiftCertificates CONFIG REQUIRED)
```

Comment:
Need SwiftCrypto here?

Reply:
Maybe? We're getting it transitively through swift-certificates, but I guess we also want to use it directly?

Reply:
We do want to use import it directly, but I don't know if that means we need `find_package` call here.

Reply:
We don't strictly need it, but I think we should have it if we have a direct dependency. Otherwise we'll break if we ever remove the swift-certificates dependency